models/google_sheets_api.py

```diff
@@ -29,14 +29,17 @@ class SheetsAPI:
     _title_height = {
         _PRICES_SHEET_NAME: 2,
         _BALANCES_SHEET_NAME: 2,
-        _ADDRESSES_SHEET_NAME: 0,
+        _ADDRESSES_SHEET_NAME: 1,
         _PORTFOLIO_SHEET_NAME: 1,
         # _PERFORMANCE_SHEET_NAME: 2,
         _DAILY_PERFORMANCE_SHEET_NAME: 3
     }
 
-    def __init__(self, sheets_id='1kFi2uGX3RZYFj76cXNE93ys3hJY49XhHtPCj_Sc5nNE', app_name='Assets Parser'):
+    def __init__(self, sheets_id, app_name='Assets Parser', secret_file=None):
+
         self._SHEETS_ID = sheets_id
+        if secret_file is not None:
+            self._CLIENT_SECRET_FILE = secret_file
         self._APPLICATION_NAME = app_name
         self._credentials = self._get_credentials()
         http = self._credentials.authorize(httplib2.Http())
```

Comment:
Let's add `cache_discovery=False` to `discovery.build()` to get rid of warning messages as suggested here: https://stackoverflow.com/a/44518587/4793743